### PR TITLE
[FEAT] Auction Items

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata/metadata.ts [hunk 1]
```diff
@@ -5679,4 +5678,20 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
+    description:
+      "With his trusty tusks and love for the deep, he'll ensure you reel in an extra fish every time",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/question_mark.png",
+    attributes: [],
+  },
+  Alba: {
+    description:
+      "With her keen instincts, she ensures you get a little extra splash in your catch. +0.5 fish!",
```
Comment (brian-man) [inline metadata/metadata.ts:5691]: Adam stated explicitly in one of the streams that a partial fish doesn't really work well.  For buds, that means that the impact is a chance of getting +1 fish.  Should this be changed to align? (50% chance of +1?)

File: src/features/island/collectibles/CollectibleCollection.tsx [hunk 2]
```diff
@@ -412,6 +414,10 @@ export const COLLECTIBLE_COMPONENTS: Record<
   "Town Sign": Sign,
   "White Crow": WhiteCrow,
   Bud: Bud,
+
+  // Fishing Season SFTs
```
Comment (brian-man) [inline src/features/island/collectibles/CollectibleCollection.tsx:418]: Consider: Use official name of the season (what the banner and ticket are called).

File: metadata/metadata.ts [hunk 3]
```diff
@@ -5840,4 +5840,68 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  Walrus: {
+    description:
+      "With his trusty tusks and love for the deep, he'll ensure you reel in an extra fish every time",
```
Comment (spencerdezartsmith) [inline metadata/metadata.ts:5845]: This is a nit but can we be consistent with the punctuation? Just need a `.` at the end of this description. Maybe just check the other to be sure they all have punctuation.

**Review by spencerdezartsmith — DISMISSED** [review]
Just a small comment. Non blocking.

File: metadata/metadata.ts [hunk 4]
```diff
@@ -5840,4 +5840,68 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
+      "With her keen instincts, she ensures you get a little extra splash in your catch. 50% chance of +1 fish!",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    // image_url: "../public/erc1155/images/479.png",
+    image_url: "../public/erc1155/images/question_mark.png",
+    attributes: [],
+  },
+  "Knowledge Crab": {
+    description:
+      "The Knowledge Crab doubles your compost effects, making your soil treasures as rich as sea plunder!",
```
Comment (brian-man) [inline metadata/metadata.ts:5863]: Is this intended to affect Rapid Root? If so... instant crops?

**Pull request merged**